tools/rna_tools/trna_mod/gatk_wrapper.xml

```diff
@@ -0,0 +1,90 @@
+<tool id="tRNAmodgatk" name="gatk_for_tRNAmod" version="0.1.1">
+  <description>GATK for tRNAmod analysis</description>
+  <requirements>
+	<requirement type="package" version="2.7">python</requirement>
+	<requirement type="package" version="1.4.1">samtools</requirement>
+	<requirement type="package" version="2.9.2">picard</requirement>
+	<requirement type="package" version="8.0.92">java-jdk</requirement>
+  </requirements>
+  <command>
+	<![CDATA[
+			 ##cleanup
+			 rm -f "${reference}.dict";
+			 rm -f "${reference}.fasta.fai";
+			 rm -f "${input}.bam.bai";
+			 rm -f "${input}.bai";
+			 ## prepare fasta index and dict and link files for correct file type
+             ln -sf "${input}" "${input}.bam";
+             ln -sf "${reference}" "${reference}.fasta";
+			 #if str($gatktool.tool) == "IndelRealigner":
+			     ln -sf "${intervals}" "${intervals}.intervals";
+			 #end if
+			 samtools faidx "${reference}.fasta" ;
+			 picard CreateSequenceDictionary R= "${reference}.fasta" O= "${reference}.dict";
+			 samtools index "${input}.bam";
+			 picard BuildBamIndex I="${input}.bam" O="${input}.bam.bai";
+			 			 
+			 ##now run gatk
+			 java -jar $__tool_directory__/GenomeAnalysisTK.jar
```

Comment:
@bgruening Hi, thanks for checking the tools. a general comment, this is jsut a collection of tools for a particular workflow, thus the wrapper for e.g. bbduk, as parameters are different from default and should not be changed by the user.
There is no way of using gatk4 as the workflow depends on gatk3.6, the bioconda solution for that is to just download the binary and link it to the bioconda dir, I can not change that.
Version 4 is also in beta and does not offer the utilities we need (yet), maybe when they release the official build at the end of september, if so I'll adapt the tools accordingly.
However, the travis build will always fail due to the gatk dependency, I have no idea how to change that and still have proper tests.